### DISCUSSION TITLE
Displaying multiple equal symbol (`=`) seems odd / different

### DISCUSSION THREAD

**Discussion by akhilerm** [ghostty-org/ghostty#4150] [Q&A] [answered]
While typing multiple `=`, ghostty terminal displays it as different symbols depending on the number of equals.

![out](https://github.com/user-attachments/assets/c18fef78-771d-4409-8fb6-4f4ff8a24ba0)

In the above attached gif, it can be see that for double equal symbol, there is no space in between the characters. But when an individual symbol is selected it shows the space. 

Similarly for three equal symbols, it is displayed differently. Again this changes when individual characters are selected. 

Is this an expected behaviour?

**Comment by JakeWharton** [comment, marked as answer]
These are ligatures. If you want to disable, see https://github.com/ghostty-org/ghostty/discussions/3323#discussioncomment-11698037, as well as the larger discussion there.
